android/java/org/chromium/chrome/browser/app/domain/NetworkModel.java

```diff
@@ -333,9 +333,8 @@ public List<NetworkInfo> stripNoBuySwapNetworks(
             List<NetworkInfo> networkInfos, BuySendSwapActivity.ActivityType type) {
         List<NetworkInfo> networkInfosFiltered = new ArrayList<>();
         for (NetworkInfo networkInfo : networkInfos) {
-            if (type == BuySendSwapActivity.ActivityType.BUY && Utils.allowBuy(networkInfo.chainId)
-                    || (type == BuySendSwapActivity.ActivityType.SWAP
-                            && Utils.allowSwap(networkInfo.chainId))) {
+            if (type == BuySendSwapActivity.ActivityType.BUY
```

Comment:
nit: can rename `stripNoBuySwapNetworks` to `stripNoBuyNetworks`

Reply:
Fixed.